### PR TITLE
Recalculate original state when resize() was called

### PR DESCRIPTION
**Pull request by kevinlaw91**
Proposed fix for #185

### CODE REVIEW

**Comment by kevinlaw91** [comment]
Might need to refactor the fix  because most of the code in this fix was borrowed from `ShadowViewport.prototype.processCTM()`.

We cannot reuse `processCTM()` in the fix because we don't want to call `setCTM()` at the end. Calling it will reset zoom and pan every time when viewport resizes.

**Comment by bumbu** [comment]
Thank you for the work. 
Will take a closer look at some point. 
Agree that we can refactor the code and have less duplication

File: src/shadow-viewport.js [hunk 1]
```diff
@@ -133,8 +136,7 @@ ShadowViewport.prototype.processCTM = function() {
   this.originalState.x = newCTM.e
   this.originalState.y = newCTM.f
 
-  // Update viewport CTM and cache zoom and pan
-  this.setCTM(newCTM);
```
Comment (kevinlaw91) [inline src/shadow-viewport.js]: This line is only used by `init()`. So i moved it out and place it after calling `processCTM()`.

Comment (kevinlaw91) [inline src/shadow-viewport.js]: This will allow reuse of `processCTM()` in `SvgPanZoom.prototype.resize()`

**Comment by kevinlaw91** [comment]
@bumbu, I have refactored the code. You can test to see if it is ready for merge. This is a fix of behavior so could be a breaking change.

**Comment by bumbu** [comment]
Thanks @kevinlaw91 
If it is a breaking change then it may be better to postpone it until next major release. 
I'm really busy now and just can't find enough time for a bigger release, but I'll try as soon as possible.

**Comment by kevinlaw91** [comment]
Thanks for your time. FYI this is just a simple bug fix for an odd behavior. I believe the old behavior wasn't suppose to be a feature, so it may not be a breaking change either. Anyway, feel free to test it out when you have time. Hope to see this get merged before next major release. :v:

**Comment by bumbu** [comment]
I tested it with different SVGs, also I paid attention to SVGs with `viewBox` attribute. All worked fine.
The changes make sense, and I agree that old behaviour shouldn't be a feature. Still some project may depend on that so I'll release it as a new minor version.

Thanks for the effort!

**Pull request merged**